components/constellation/constellation.rs

```diff
@@ -532,13 +532,15 @@ pub struct InitialConstellationState {
 struct WebDriverData {
     load_channel: Option<(PipelineId, IpcSender<WebDriverLoadStatus>)>,
     resize_channel: Option<IpcSender<Size2D<f32, CSSPixel>>>,
+    sync_channel: Option<IpcSender<WebDriverCommandResponse>>,
 }
 
 impl WebDriverData {
     fn new() -> WebDriverData {
         WebDriverData {
             load_channel: None,
             resize_channel: None,
+            sync_channel: None,
```

Comment:
Is the only purpose of `sync_channel` to reply to actions?

If so, please rename it and add comments as such.